### PR TITLE
Enable bazel  rust_test for authorization logic. 

### PR DESCRIPTION
**Pull request by bgogul**

### CODE REVIEW

File: rust/tools/authorization-logic/src/utils/utils.rs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+// pub fn create_output_file(path: &str) -> Result<File, std::io::Error> {
+//     File::create(&get_resolved_output_path(path))
+// }
+
+// pub fn open_input_file(path: &str) -> Result<File, std::io::Error> {
+//     println!("Opening {} for input", get_resolved_path(path).to_string());
+//     File::open(&get_resolved_path(path))
+// }
```
Comment (markww) [inline rust/tools/authorization-logic/src/utils/utils.rs:57]: Should these have been removed?

File: rust/tools/authorization-logic/BUILD [hunk 2]
```diff
@@ -16,7 +16,8 @@
 #-------------------------------------------------------------------------------
 package(default_visibility = ["//visibility:public"])
 
-load("@rules_rust//rust:rust.bzl", "rust_binary", "rust_library")
+#load("@rules_rust//rust:defs.bzl", "rust_binary", "rust_library", "rust_static_library", "rust_test")
```
Comment (markww) [inline rust/tools/authorization-logic/BUILD:19]: Should this have been removed?

**Review by markww — APPROVED** [review]
Looks good, other than some possibly-stale comments.

**Comment by markww** [comment]
Whoops, didn't see this was a draft, sorry for the early review.

File: rust/tools/authorization-logic/src/utils/utils.rs (hunk 1 above)
Comment (bgogul) [reply rust/tools/authorization-logic/src/utils/utils.rs]: It was in the draft. These are removed.

File: rust/tools/authorization-logic/BUILD (hunk 2 above)
Comment (bgogul) [reply rust/tools/authorization-logic/BUILD]: Removed from draft.

**Comment by bgogul** [comment]
@markww this is ready for review. 

@aferr can you take a look as well?

File: rust/tools/authorization-logic/src/utils/utils.rs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            "{}/{}", tmp_dir, input_path);
+        println!("Linking {} as {}", resolved_input_path, resolved_link_path);
+        std::process::Command::new("ln")
+            .arg("-s")
+            .arg(resolved_input_path)
+            .arg(resolved_link_path)
+            .spawn()
+            .expect("had an error when trying to start ls")
+            .wait_with_output()
+            .expect("had an error in output of ls");
```
Comment (markww) [inline rust/tools/authorization-logic/src/utils/utils.rs:57]: It looks like you're running `ln` but your error messages say `ls`.

**Review by markww — APPROVED** [review]
LGTM other than a few easy-to-fix issues,

File: gcb/presubmit.sh [hunk 4]
```diff
@@ -101,8 +101,9 @@ bazel_run build \
   //third_party/arcs/examples:consume \
   //third_party/arcs/proto:manifest_cc_proto \
   //src/analysis/souffle/tests/arcs_manifest_tests_todo/... \
-  //src/analysis/souffle/examples/... \
-  //rust/tools/authorization-logic:authorization_logic
+  //src/analysis/souffle/examples/...
 
 # Run all the bazel tests (not cargo).
```
[the remark this answers is not in the document]
Comment (bgogul) [reply gcb/presubmit.sh]: done.

File: rust/tools/authorization-logic/src/utils/utils.rs (hunk 3 above)
Comment (bgogul) [reply rust/tools/authorization-logic/src/utils/utils.rs]: AH, good catch. Fixed now.

**Review by aferr — APPROVED** [review]
LGTM

**Pull request merged**